cli/parser/options/flag_form/flag_form.go

```diff
@@ -0,0 +1,69 @@
+package flag_form
+
+type Form byte
```

Comment:
Can you add a comment about what this represents?

One question I had also was whether `Standard` can only represent `--name=true` or if it can also represent `--name=false`, or if that would fall under "Negative". Might be good to clear that up here.